httpx/instrument_test.go

```diff
@@ -0,0 +1,50 @@
+package httpx_test
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"testing"
+	"time"
+
+	"github.com/socialpoint-labs/bsk/httpx"
+	"github.com/socialpoint-labs/bsk/metrics"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestInstrument_RequestsDuration(t *testing.T) {
+	assert := assert.New(t)
+
+	for _, testCase := range []struct {
+		waitTime     time.Duration
+		deltaTime    time.Duration
+		tags         metrics.Tags
+		expectedTags int
```

Comment:
could you add some headers to the test case? so it's obvius headers are not erased by the delegator